lib/actions/api.js

```diff
@@ -150,6 +150,26 @@ function getJsonAndCheckResponse (res) {
   return res.json()
 }
 
+/**
+ * Gets a non-blank userId query parameter if all the following applies:
+ * - The user is logged in (otpState.user.loggedInUser is not null),
+ * - The user had an id attribute (should be always true),
+ * - The user has consented to storeTripHistory.
+ */
+function makeUserIdParam (otpState) {
```

Comment:
Can we rename this to something like `makeStoreTripHistoryParams` to indicate that is the primary motivation for adding the user id?

Reply:
The user id param will be removed per recent change to middleware.